t/evaluator/evaluate-oom-job.t

```diff
@@ -0,0 +1,33 @@
+use strict;
+use warnings;
+use Setup;
+use Test2::V0;
+use Hydra::Helper::Exec;
+
+my ($systemdrRes) = captureStdoutStderr(3, (
+   "systemd-run", "--user", "--collect", "--scope", "--property", "MemoryMax=25M", "--",
+   "true"
+));
```

Comment:
Looks like we need to do this to handle the case that systemd-run isn't available:

```suggestion
exec {
  my ($systemdrRes) = captureStdoutStderr(3, (
     "systemd-run", "--user", "--collect", "--scope", "--property", "MemoryMax=25M", "--",
     "true"
  ));
} or do {
  skip_all("systemd-run does not work in this environment");
}
```